app/assets/builds/tranzito_utils-compiled.css

```diff
@@ -142,7 +142,7 @@ input[type=radio] {
   max-width: 100%;
   vertical-align: middle;
 }
-.custom-time-selection .btn-success {
+.custom-time-selection #updatePeriodSelectCustom {
```

Comment:
We should use `.btn-primary`  instead of `#updatePeriodSelectCustom` as we were using the  `.btn-success`
or is there any reason we are doing it this way?

Reply:
I did it In case we decided to switch the button type again, so that we didn't need to update it. We don't actually want to attach it to the button type, but to the specific button.